pymeasure/instruments/__init__.py

```diff
@@ -22,56 +22,5 @@
 # THE SOFTWARE.
 #
 
-from ..errors import RangeError, RangeException
 from .channel import Channel
 from .instrument import Instrument
-from .resources import list_resources
```

Comment:
Why remove that one? That seems quite useful to have at the `pymeasure.instruments` layer. I thought that the instruments create the big memory footprint?

Reply:
In short I removed this import because it isn't required for all instruments and so it shouldn't be included in the init.py. Functionally there is no difference at the moment, because `Instrument` imports `VISAAdapter` thus `pyvisa`, but as per the `SCPIMixin` class that can change https://github.com/pymeasure/pymeasure/pull/905 . 

That's my reasoning, but if you think it should be included for other reasons I can revert this change.

Reply:
>  because it isn't required for all instruments and so it shouldn't be included in the init.py. 

I'm not sure I follow this line of thought. We should then also remove `Channel`, because that's also not required for all instruments.
I'm thinking that this is a useful convenience feature for users in need of troubleshooting, so it's good if it can be easily discovered/accessed. Might be that others disagree? @bmoneke ?

Reply:
I've never used it, so at least for me, it is not that important, to have it at that position.
But other use cases need it more often.

It might be useful, if you use it more dynamically, plugging devices in and out.

Reply:
I'm open for keeping it in, if anyone deems it useful. As you like to keep it, let us keep it in the init file.

Reply:
Sounds good, I reverted this change in both docs and instruments/__init__.py